### PR TITLE
Add the new spinner

### PR DESCRIPTION
**Pull request by rubene**
:art:
* Move the spinner css from automotive app because this will be share
  across all apps.

### CODE REVIEW

File: app/assets/stylesheets/layout_components/spinner.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.spinner{
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.double-bounce1, .double-bounce2{
```
Comment (kaytcampbell) [inline app/assets/stylesheets/layout_components/spinner.scss]: the second class should be on it's own line

File: app/assets/stylesheets/layout_components/spinner.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  background-color: #FFF;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/layout_components/spinner.scss]: can we put $white here instead of #fff

**Comment by kaytcampbell** [comment]
:hankey:

**Comment by rubene** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**